apm-sniffer/apm-sdk-plugin/rocketMQ-client-java-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/rocketMQ/client/java/v5/MessageListenerInterceptor.java

```diff
@@ -37,30 +38,38 @@
 public class MessageListenerInterceptor implements InstanceMethodsAroundInterceptor {
 
     public static final String CONSUMER_OPERATION_NAME_PREFIX = "RocketMQ/";
+    public static final Gson GSON = new Gson();
 
     @Override
-    public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes, MethodInterceptResult result) throws Throwable {
+    public void beforeMethod(EnhancedInstance objInst,
+                             Method method,
+                             Object[] allArguments,
+                             Class<?>[] argumentsTypes,
+                             MethodInterceptResult result) throws Throwable {
         MessageView messageView = (MessageView) allArguments[0];
 
         ContextCarrier contextCarrier = getContextCarrierFromMessage(messageView);
 
         AbstractSpan span = ContextManager.createEntrySpan(CONSUMER_OPERATION_NAME_PREFIX + messageView.getTopic()
-                + "/Consumer", contextCarrier);
+                                                               + "/Consumer", contextCarrier);
         Tags.MQ_TOPIC.set(span, messageView.getTopic());
-
+        span.tag(Tags.ofKey("mq.message.id"), messageView.getMessageId().toString());
```

Comment:
`Tags#ofKey` is very expensive. The tag should be a static field, rather than created repeatedly.